scripts/openmetadata-tagging-project/README.md

```diff
@@ -0,0 +1,146 @@
+# OpenMetadata Tagging Project
+
+This script is used to tag assets in OpenMetadata with an app tag by using the fully qualified name (FQN) in OpenMetadata with a list returned from the SQL script which queries the ER Studio database. If a match is found a tag is applied to the asset to link it to an application. The script has a `--dry-run` option to test before making changes to OpenMetadata.
+
+## Folder Structure
+```
+   openmetadata-tagging-project/
+   ├─ .env.example
+   ├─ .gitignore
+   ├─ README.md
+   ├─ config/
+   │  ├─ asset_ownership_er_studio.sql
+   │  └─ openmetadata_config.json.example
+   ├─ data/
+   │  └─ <csv will generate here>
+   ├─ docs/
+   │  └─ DEVELOPMENT.md
+   ├─ logs/
+   │  └─ <log file will generate here>
+   ├─ src/
+   │  ├─ __init__.py
+   │  ├─ db_connection_cx.py
+   │  ├─ fetch_openmetadata_fqns.py
+   │  ├─ main.py
+   │  └─ openmetadata_table_list_processor.py
+   └─ tests/
+      └─ test_main.py
+```
+
+## Features
+
+- Applies tags to OpenMetadata tables based on application ownership
+- Supports a `--dry-run` mode for testing
+- Includes error handling and retry logic for network issues
+
+## Prerequisites
+
+- Python 3.8.10
+- Required Python packages (refer to a requirements.txt or pipfile)
+- Access to OpenMetadata instance
+- Necessary permissions and credentials for OpenMetadata and Oracle database
+
+## Installation
+
+### Steps to set up the project outside of a development environment:
+
+1. Clone the repository
+   ```
+   git clone https://github.com/yourusername/openmetadata-tagging-project.git
+   cd openmetadata-tagging-project
+   ```
+
+2. Install dependencies:
+   ```
+   pip install -r requirements.txt`
```

Comment:
We're missing the requirements.txt file

Reply:
Totally missed those in the initial commit. Included pipfiles too.